src/openMVG/sfm/sfm_data_io_gt.hpp

```diff
@@ -119,35 +126,47 @@ static bool read_Strecha_Camera(const std::string & camName, Pinhole_Intrinsic &
 
 /**
 @brief Reads a set of Pinhole Cameras and its poses from a ground truth dataset.
-@param[in] functorPointer, to the function which can handle the trajectory format. Example: &read_openMVG_Camera
 @param[in] sGTPath, the directory where the camera files are located.
```

Comment:
I think 2 is better. if u want you can add two optional input parameters with the names of the folders for ground truth and images, which by default are set to  `gt_dense_cameras` and  `images`, something like
```c++
bool readGt(const std::string basePath, 
             SfM_Data & sfm_data,
             const std::string imgDirectoryName = "images",
             const std::string gtDirectoryName = "gt_dense_cameras");
```
In case update the doc then